spec/features/state_file/review_page_spec.rb

```diff
@@ -108,56 +120,6 @@
     end
   end
 
-  context "NC" do
-    it "allows user to navigate to unemployment review page, edit an unemployment 1099g form, and then navigate back to final review page", required_schema: "nc" do
-      state_code = "nc"
-      set_up_intake_and_associated_records(state_code)
-
-      intake = StateFile::StateInformationService.intake_class(state_code).last
-
-      visit "/questions/#{state_code}-review"
-
-      # Final review page
-      expect(page).to have_text I18n.t("state_file.questions.shared.abstract_review_header.title")
-      within "#income-info" do
-        click_on I18n.t("general.edit")
-      end
-
-      # income-info edit navigates to unemployment index page
-      expect(page).to have_text(I18n.t('state_file.questions.unemployment.index.1099_label', name: intake.primary.full_name))
-      click_on I18n.t("general.continue")
-
-      # Back on final review page
-      expect(page).to have_text I18n.t("state_file.questions.shared.abstract_review_header.title")
-      within "#income-info" do
-        click_on I18n.t("general.edit")
-      end
-      click_on I18n.t("general.edit")
-
-      # 1099G edit page
-      expect(page).to have_text strip_html_tags(I18n.t("state_file.questions.unemployment.edit.title", count: intake.filer_count, year: MultiTenantService.statefile.current_tax_year))
-      fill_in strip_html_tags(I18n.t("state_file.questions.unemployment.edit.payer_name")), with: "beepboop"
-      click_on I18n.t("general.continue")
-
-      # takes them to the 1099G index page first
-      expect(page).to have_text strip_html_tags(I18n.t("state_file.questions.unemployment.index.lets_review"))
-
-      # edit a 1099G (there's only one)
-      click_on I18n.t("general.edit")
-      click_on I18n.t("general.continue")
-
-      # back on index page
-      expect(page).to have_text strip_html_tags(I18n.t("state_file.questions.unemployment.index.lets_review"))
-
-      # delete a 1099G (there's only one)
-      recipient_name = intake.state_file1099_gs.last.recipient_name
-      click_on I18n.t("general.delete")
-      # redirects to new because there are no 1099Gs left, need to select "no" in order to continue
-      expect(page).to have_text I18n.t("state_file.questions.unemployment.destroy.removed", name: recipient_name)
-      choose I18n.t("general.negative")
-    end
-  end
```

Comment:
are we losing coverage of tests here where we test if we can edit/delete a 1099G? (or was that only a functionality that NC review page had?)

Reply:
No, this was (mostly) duplicative of the code in `edit_unemployment` in this file, which executes for every state except NJ (which doesn't allow them to be edited)

Reply:
i missed that! ty